README.md

```diff
@@ -25,7 +25,8 @@ Documentation on these components and how to use them can be found on [Amazon Ch
 
 [Amazon Chime SDK for JavaScript Documentation](https://aws.github.io/amazon-chime-sdk-js/#amazon-chime-sdk-for-javascript)
 
-[Migration to V2](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/migration-to-v2--page)
+[Chime SDK UI Component Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/)
+[Migration from V1 to V2](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/migration-to-v2--page)
```

Comment:
We have already updated the README, @ldai1 could you please check once if these are now needed?

Reply:
You are right. I checked the commit: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/450 and it already removed the [Chime SDK UI Component Library Documentation]. Will remove it in this housekeeping.

Reply:
Yeah right, Vidya also added some steps around installing and running the library. It would be great to just compare and add those back and modify few links to be on new lines if not. Thanks for taking care of this.